tests/behat/features/file-download.d7.feature

```diff
@@ -0,0 +1,36 @@
+@d7
+Feature: Check that FileDownloadTrait for D7 works
+
+  Background:
+    Given I am logged in as a user with the "administrator" role
+    And article content:
+      | title                | field_file                                                   |
+      | [TEST] document page | file;example_files/example_document.txt;example_document.txt |
+    And article content:
+      | title           | field_file                                             |
+      | [TEST] zip page | file;example_files/example_files.zip;example_files.zip |
+
+  @api @download
+  Scenario: Assert that a file can be downloaded
+    And I download file from "/example_files/example_document.txt"
+
+  @api
+  Scenario: Assert that a file can be downloaded by existing link
+    When I visit article "[TEST] document page"
+    Then I see download "Download example_document.txt" link "present"
+    Then I download file from link "Download example_document.txt"
+    And downloaded file contains:
```

Comment:
Please add an additional step `And downloaded file does not contain:` and a test for it